app/models/commodity.rb

```diff
@@ -12,10 +12,10 @@ def description
 
   def response
     @response ||= begin
-                    commodity = Uktt::Commodity.new(options)
-                    commodity.retrieve
-                    commodity.response
-                  end
+      commodity = Uktt::Commodity.new(options)
```

Comment:
The linter told me to do this :man_shrugging:

Reply:
weird...